lib/ex_webrtc/peer_connection.ex

```diff
@@ -192,26 +192,34 @@ defmodule ExWebRTC.PeerConnection do
   @impl true
   def handle_call({:create_answer, _options}, _from, state)
       when state.signaling_state in [:have_remote_offer, :have_local_pranswer] do
+    {:offer, remote_offer} = state.pending_remote_desc
+
     {:ok, ice_ufrag, ice_pwd} = ICEAgent.get_local_credentials(state.ice_agent)
     {:ok, dtls_fingerprint} = ExDTLS.get_cert_fingerprint(state.dtls_client)
 
     answer = %ExSDP{ExSDP.new() | timing: %ExSDP.Timing{start_time: 0, stop_time: 0}}
 
+    answer =
+      case ExSDP.get_attribute(remote_offer, :ice_options) do
+        {:ice_options, "trickle"} = attr -> ExSDP.add_attribute(answer, attr)
+        _other -> answer
+      end
```

Comment:
Just FYI we don't support anything other than `trickle` ice so this attribute has to always be in offer/answer